wrappers/services_test.go

```diff
@@ -725,8 +725,6 @@ func (s *servicesTestSuite) TestAddSnapMultiServicesFailCreateCleanup(c *C) {
 	if len(s.sysdLog) > 0 {
 		// the second service failed validation
 		c.Check(s.sysdLog, DeepEquals, [][]string{
-			{"--root", dirs.GlobalRootDir, "enable", "snap.hello-snap.svc1.service"},
```

Comment:
I'm puzzled why this test changed.

Reply:
This is because in the old implementation we had a loop that:
-  **validates** service definition and writes unit file for the service.
- enables it
- moves onto next service

If one of them failed, we had to disable those that were  enabled so far.

With the new helper we write all units first in one go, then enable all in one go separately if validation was ok for all services.